db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
+      column :slug, :text, null: false, unique: true
+      column :title, :text, null: false
+      column :body, :text, null: false
+      column :published, :boolean, default: false, null: false
```

Comment:
@solnic Oh yes, great suggestion. I've just made this change in https://github.com/dry-rb/dry-web-blog/pull/1/commits/9364dfa22c3caa78b2eb61a5ef5f2c52f999f372, but I note there's no `Articles#published`, but rather `Articles#by_published` – is that what you meant?